storage/tianmu/core/tianmu_attr_exqp.cpp

```diff
@@ -810,6 +810,18 @@ void TianmuAttr::EvaluatePack_BetweenString_UTF(MIUpdatingIterator &mit, int dim
       return;
     }
   }
+
+  auto check_item_digital = ([&](CQTerm &val) {
+    return ((nullptr != val.item) &&
+            (Item::Type::INT_ITEM == val.item->type() || (Item::Type::DECIMAL_ITEM == val.item->type()) ||
```

Comment:
`const_val ==  var` is an old fashion, in ok in old compiler, but it's not necessary for a new compiler.